base/nsjail-docker/files/drop_privs

```diff
@@ -2,4 +2,4 @@
 
 # There are two copies of this file in the nsjail and healthcheck base images.
 
-exec setpriv --init-groups --reset-env --reuid user --regid user --inh-caps=-all -- $@
+exec setpriv --init-groups --reset-env --reuid user --regid user --inh-caps=-all -- "$@"
```

Comment:
Why do you need to quote it? Doesn't that make command line arguments the name of the command?

Reply:
"$@" will be replaced with "$1" "$2" ...
It's magic :)

Reply:
cool!